tests/lib/collectors/gcloud.py

```diff
@@ -51,16 +51,19 @@ def testInitialization(self):
     self.assertIsNotNone(gcloud_collector)
 
   # pylint: disable=invalid-name,line-too-long
+  @mock.patch('libcloudforensics.providers.gcp.internal.compute.GoogleCloudCompute.GetInstance')
   @mock.patch('libcloudforensics.providers.gcp.internal.compute_base_resource.GoogleComputeBaseResource.AddLabels')
   @mock.patch('libcloudforensics.providers.gcp.internal.compute_base_resource.GoogleComputeBaseResource')
   @mock.patch('libcloudforensics.providers.gcp.forensics.StartAnalysisVm')
   def testSetUp(self,
                 mock_StartAnalysisVm,
                 mock_GoogleComputeBaseResource,
-                mock_AddLabels):
+                mock_AddLabels,
+                mock_TargetInstance):
```

Comment:
nit: to be consistent with other mocks, can we name the mock variables after the function they're mocking? In this case

```suggestion
                mock_GetInstance):
```

Same for the variable in Line 117